### DISCUSSION TITLE
What is the status of this project?

### DISCUSSION THREAD

**Discussion by oparada1988** [muesli/obs-cli#72] [General]
I know asking for an ETA is a no, no. But I just wanted to know the status of the project? Last update will be a year pretty soon.

**Comment by muesli** [comment]
Currently waiting on a few updates for the OBS websockets v5 API, as I try to avoid breaking compatibility with previous versions.

**Reply by nicoaarnio** [reply]
Is this project still alive?

**Reply by muesli** [reply]
I don't think anything has changed since my last reply. So yeah, alive, but not migrated to v5.

**Comment by oparada1988** [comment]
Thanks for the prompt reply! I have zero skills in programming, etc. I do want to learn though, so if you could share some documentation on how I can start contributing I would be so thankful!
